spotify_player/src/ui/playback.rs

```diff
@@ -184,7 +184,11 @@ fn render_playback_text(
                     } else {
                         &state.app_config.play_icon
                     },
-                    track.name,
+                    if track.explicit {
+                        format!("{} (Explicit)", track.name)
+                    } else {
+                        track.name.clone()
+                    }
```

Comment:
The `track` variable here is an `rspotify::FullTrack`, not a `spotify_player::Track`, so I couldn't reuse the display_name method